File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.UserService.Models/business-unit_item.ts


import { IBusinessUnit_Item } from './ibusiness-unit_item';

export class BusinessUnit_Item implements IBusinessUnit_Item {
  
	public BusinessUnitID: number = 0;
	public BusinessUnitName: string = "";
	public Acronym: string = "";
	public BusinessParentID?: number;
	public PostID?: number;
	public UnitLibraryUnitID?: number;
	public VettingPrefix: string = "";
  
}

File: GTTS/INL.UserService.Models/BusinessUnit_Item.cs
namespace INL.UserService.Models
{
	public class BusinessUnit_Item : IBusinessUnit_Item
	{
		public int BusinessUnitID { get; set; }
		public string BusinessUnitName { get; set; }
		public string Acronym { get; set; }
		public int? BusinessParentID { get; set; }
		public int? PostID { get; set; }
		public long? UnitLibraryUnitID { get; set; }
		public string VettingPrefix { get; set; }
	}
}
